src/jinja2/filters.py

```diff
@@ -6,6 +6,13 @@
 from collections import namedtuple
 from itertools import chain
 from itertools import groupby
+from typing import Any
+from typing import Iterable
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import TypeVar
+from typing import Union
```

Comment:
Is this the prefered code style? For so many imports it seems a lot more intuitive to put these all on a single line (`from typing import Any, Iterable, List, Optional, Sequence, TypeVar, Union`)

Reply:
This is the way the pre commits formatted it. Not my preference at all.